### ISSUE TITLE
prefer_inline, false positive?

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1867]
something like this data definition should probably be skipped?

```abap
FORM bar.
  DATA lv_text TYPE c LENGTH 200.
  lv_text = 'foo'.
ENDFORM.
```

![image](https://user-images.githubusercontent.com/5888506/111582679-79618380-87bb-11eb-80fc-b7b90d25bc34.png)

**Comment by larshp** [comment]
still an issue
